README.md

```diff
@@ -35,3 +35,95 @@ console.log(binaryPacket.toString('hex'))
 
 const jsonPacket = packet.deserializeIlpPayment(binaryPacket)
 ```
+
+### Serialize/deserialize ILQP Quote Requests/Responses
+
+#### QuoteLiquidityRequest
+
+```js
+const packet = require('ilp-packet')
+
+const binary = packet.serializeIlqpLiquidityRequest({
+  requestId: 'c217449a-69e9-4a08-9575-6e7eafa818c0',
+  destinationAccount: 'example.nexus.bob',
```

Comment:
> One account would mess up that caching mechanism because it would be technically incorrect to say that all accounts under that prefix share the same liquidity curve, even if you know that 10,000 do and 1 doesn't.

Correct. As the last connector in the chain, I have to check to make sure I have a **local** route *R* that matches the requested address. If so, the `appliesToPrefix` is the shortest prefix that **uniquely** matches *R*. If I have other routes that match *R* I have to add characters from the queried ILP address back until it no longer matches any of those other routes.

It's unfortunate - but unavoidable - that having one counterexample on a ledger nullifies the ability to have a common cached route. You could design your address space to avoid that case (e.g. with the `main` address segment we were talking about before.)

I'd be against making the appliesTo feature more complex (boolean list of applicable prefixes.) To add that feature later, we can add an optional field using the extensibility mechanism once we have a higher tolerance for complexity and more actual implementation experience.